frontend/crd/src/app/api-documentations/api.json

```diff
@@ -0,0 +1,489 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "title": "Sample API",
+    "description": "API description in Markdown.",
+    "version": "1.0.0"
+  },
+  "host": "localhost:8080",
+  "basePath": "/api",
+  "schemes": ["http"],
+  "paths": {
+    "/dashboard-events": {
+      "get": {
+        "summary": "Returns the selection of events to display on the dashboard carousel",
+        "description": "Optional extended description in Markdown.",
+        "tags": ["dashboard"],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "studentID",
+            "required": true,
+            "schema": {
+              "type": "integer"
+            },
+            "description": "Only return events relevant to this student"
+          },
+          {
+            "in": "query",
+            "name": "pageNum",
+            "required": false,
+            "schema": {
+              "type": "integer",
+              "default": 1,
+              "minimum": 1
+            },
+            "description": "Which page of events to display"
+          }
+        ],
+        "produces": ["application/json"],
+        "responses": {
```

Comment:
can we add some schemas to the responses? see https://swagger.io/docs/specification/describing-responses/